.lgtm

```diff
@@ -0,0 +1,2 @@
+pattern = "(?)LGTM"
+self_approval_off = true
```

Comment:
Currently, each PR requires _two_ reviews from one of the people in the file below. This line prevents maintainers from approving themselves.